corehq/apps/app_manager/tests/data/v2_diffs/partials/case_config_shared.html.diff.txt

```diff
@@ -123,14 +109,44 @@
 -                <td class="col-sm-5">
 -                    <div data-bind="template: {
 -                        name: 'case-config:case-properties:property',
--                        data: {property: $data, suggestedProperties: case_transaction.suggestedSaveProperties}
+-                        data: {property: $data, suggestedProperties: case_transaction.suggestedSaveProperties},
 -                    }"></div>
 -                    <p class="help-block" data-bind="html: validate, visible: validate"></p>
+-                    <!-- ko if: $data.description -->
+-                    <inline-edit params="
+-                        value: $data.description,
+-                        rows: 3,
+-                        cols: 30,
+-                        placeholder: '{% trans "Click here to add a description" %}',
+-                        url: '{% url "update_property_description" domain %}',
+-                        saveParams: {'caseType': $data.caseType, 'name':  $data.key},
+-                        saveValueName: 'description',
+-                        errorMessage: '{% trans "Error updating description. Please try again." %}',
+-                        readOnlyClass: 'property-description',
+-                        readOnlyAttrs: {'data-content': $data.description},
+-                        afterRenderFunc: $root.makePopover,
+-                    "></inline-edit>
+-                    <!-- /ko -->
```

Comment:
it looks like its added further down in the diff, i think its just a weird artifact of how the diff algorithm worked